filebeat/config/config.go

```diff
@@ -21,6 +21,7 @@ const (
 type Config struct {
 	Prospectors      []*common.Config `config:"prospectors"`
 	RegistryFile     string           `config:"registry_file"`
+	RegistryFlush    time.Duration    `config:"registry_flush"`
```

Comment:
Would prefer `registry.flush` and `registry.file` but too late to change I think.